packages/listview/src/components/BatchActionsBar/__test__/batchActionsBar.spec.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import BatchActionsBar from '..';
+import { StyledLabel } from '../styled';
+
+describe('<BatchActionsBar />', () => {
+    it('should render an span with the label', () => {
+        const component = mount(<BatchActionsBar label="Rides Selected" isVisible />);
+        const span = component.find(StyledLabel);
+        expect(span.text().includes('Rides Selected')).toBe(true);
+    });
+
+    it('should render an span with the itemsLength', () => {
+        const component = mount(<BatchActionsBar itemsLength={253} isVisible />);
+        const badge = component.find('Badge');
+        const span = badge.find('span');
```

Comment:
here simply check for the prop label in the Badge component have the itemsLength passed